libbpf-tools/memleak.bpf.c

```diff
@@ -0,0 +1,374 @@
+// SPDX-License-Identifier: GPL-2.0
+// Copyright (c) 2022 Jackie Dinh
+//
```

Comment:
Don't use C++ style comments.

Reply:
Fix in commin [2beb289e](https://github.com/iovisor/bcc/pull/3828/commits/2beb289e98d651bddc2127299c5627b5d6149fce).